care/facility/api/serializers/patient.py

```diff
@@ -509,6 +525,23 @@ class PatientNotesSerializer(serializers.ModelSerializer):
     thread = serializers.ChoiceField(
         choices=PatientNoteThreadChoices, required=False, allow_null=False
     )
+    reply_to = ExternalIdSerializerField(
+        queryset=PatientNotes.objects.all(),
+        write_only=True,
+        required=False,
+        allow_null=True,
+    )
+    reply_to_object = ReplyToPatientNoteSerializer(source="reply_to", read_only=True)
+    files = serializers.SerializerMethodField()
+    replies = ReplyToPatientNoteSerializer(many=True, read_only=True)
+
+    def get_files(self, obj):
+        return FileUpload.objects.filter(
```

Comment:
improvement: instead of doing this, save the files in a json array which will have the file name and reference to the file

make sure to use json schema for the object

Reply:
serialized to only return necessary fields for now